README.md

```diff
@@ -45,6 +50,8 @@ Arguments:
   arguments passed to wp_insert_post or wp_update_post. For deletes and
   undeletes, args['ID'] contains the post id to be (un)deleted. Null for queries.
 
+`$default`: The default endpoint location used by the plugin. The Authentication and Token Endpoints default to IndieAuth, the Micropub endpoint to the /?micropub=endpoint
```

Comment:
minor nits:
* please link IndieAuth (ie `[IndieAuth](https://indieauth.com/)`)
* _the /?micropub=endpoint_ => _\`/?micropub=endpoint\`_.

also in readme.txt.